tests/unit/fauxdbi.py

```diff
@@ -350,12 +350,7 @@ def _get_field(
         if columns:
             custom = columns.get(name)
             if custom:
-                return self._get_field(
-                    # We express the kwargs this way to allow custom
-                    # values to overrise name, type and nonnull, if
-                    # necessary.
-                    **dict(name=name, type=type, notnull=notnull, **custom)
-                )
+                return self._get_field(name=name, type=type, notnull=notnull, **custom)
```

Comment:
If I understood correctly, `custom`, by design, can never contain `name`, `type`, or `notnull` keys?

Reply:
You probably know better. :)

If we want to use overriding, the code needs to be changed to avoid errors (duplicate keyword arguments), otherwise we can leave it as is.

Reply:
I removed the overriding.  If we need it later, we can add it then.